.github/workflows/ci.yml

```
@@ -147,12 +145,8 @@ jobs:
     name: fmt check
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@master
```

The recommended way is it to use `@stable`.

```suggestion
      - uses: dtolnay/rust-toolchain@stable
```

---

will open a PR for this one

---

https://github.com/tauri-apps/tao/pull/957